delta-lake/src/integration/java/org/apache/iceberg/delta/TestSnapshotDeltaLakeTable.java

```diff
@@ -74,31 +72,22 @@ public class TestSnapshotDeltaLakeTable extends SparkDeltaLakeSnapshotTestBase {
   private static final String NAMESPACE = "delta_conversion_test";
   private static final String defaultSparkCatalog = "spark_catalog";
   private static final String icebergCatalogName = "iceberg_hive";
+  private static final Map<String, String> config =
+      ImmutableMap.of(
+          "type", "hive",
+          "default-namespace", "default",
+          "parquet-enabled", "true",
+          "cache-enabled",
+              "false" // Spark will delete tables using v1, leaving the cache out of sync
+          );
   private static Dataset<Row> typeTestDataFrame;
   private static Dataset<Row> nestedDataFrame;
 
-  static Stream<Arguments> parameters() {
-    return Stream.of(
-        Arguments.of(
-            icebergCatalogName,
-            SparkCatalog.class.getName(),
-            ImmutableMap.of(
-                "type",
-                "hive",
-                "default-namespace",
-                "default",
-                "parquet-enabled",
-                "true",
-                "cache-enabled",
-                "false" // Spark will delete tables using v1, leaving the cache out of sync
-                )));
-  }
-
-  @TempDir private Path temp;
+  @TempDir private File tempA;
+  @TempDir private File tempB;
 
-  public TestSnapshotDeltaLakeTable(
-      String catalogName, String implementation, Map<String, String> config) {
-    super(catalogName, implementation, config);
+  public TestSnapshotDeltaLakeTable() {
+    super(icebergCatalogName, SparkCatalog.class.getName(), config);
```

Comment:
Junit5 cannot parametrize class creation. Since we only have one set of parameters in junit4 version, I think we can directly feed them into super class.